devices/xiaomi-beryllium/default.nix

```diff
@@ -0,0 +1,25 @@
+{ config, lib, pkgs, ... }:
+
+{
+  imports = [
+    ../families/sdm845-mainline
+  ];
+
+  mobile.device.name = "xiaomi-beryllium-tianma";
```

Comment:
for some phones this is `iaomi-beryllium-ebbg` instead of  `xiaomi-beryllium-tianma`.
I suppose a switch needs to be added (or perhaps just 2 different device folders)

Reply:
[closing discussion, moving to https://github.com/NixOS/mobile-nixos/pull/560#discussion_r1047814472 which was authored at the same time]